MM2SpaceCenter.roboFontExt/lib/MM2SpaceCenter.py

```diff
@@ -15,139 +15,147 @@
 from defconAppKit.windows.baseWindow import BaseWindowController
 from mojo.events import addObserver, removeObserver, postEvent
 from vanilla import *
+from lib.UI.integerEditText import NumberEditText
 from mojo.extensions import *
 import codecs
 
 class MM2SpaceCenter:
     
     def activateModule(self):
         addObserver(self, "MMPairChangedObserver", "MetricsMachine.currentPairChanged")
+        
         print ('MM2SpaceCenter activated')
 
+
     def deactivateModule(self, sender):
         removeObserver(self, "MetricsMachine.currentPairChanged")
+        removeObserver(self, "FutureProof.currentPairChanged")
+
         print ('MM2SpaceCenter deactivated')
 
-    def __init__(self, ):
-        self.font = metricsMachine.CurrentFont()
-        self.pair = metricsMachine.GetCurrentPair()
-        #self.wordlistPath = wordlistPath
-        
-        
-        
-        
-        leftMargin = 10
-        topMargin = 5
-        yPos = 0 
-        lineHeight = 20
-        
-        yPos += topMargin
-        
-        self.messageText = 'MM2SpaceCenter activated 😎'
 
+    def __init__(self, ):
+        #self.font = metricsMachine.CurrentFont()
+        #self.pair = metricsMachine.GetCurrentPair()
 
+        self.font = CurrentFont()
+        try:
+            self.pair = metricsMachine.GetCurrentPair() 
+        except:
+             self.pair = ('A', 'V')
+        
+        leftMargin = 11
+        topMargin = 0
+        lineHeight = 18
+        columnWidth = 90  
+        second_col_x = columnWidth + leftMargin + 10
 
+        w_w = 200
         
+        self.messageText = '😎 Activated'
```

Comment:
Title bar already said the name of the extension, so I figured some brevity here would be nice.